csp_billing_adapter/adapter.py

```diff
@@ -215,8 +215,14 @@ def main() -> None:
         )
 
         initial_adapter_setup(pm.hook, config, log)
+        startup = True
 
         while True:
+            if startup:
```

Comment:
So while this avoids the initial error being reported caused by racing startups of the csp_billing_adapter and the application, it does mean that we won't attempt to validate the billing API access on the first cycle, so if there are any real errors on that front, they will take a querying interval to be detected...

We can however get this change in and address that scenario is a follow on PR?

Reply:
That wouldn't be checked anyway in the current code. Adapter would't do an empty meter until the second hour because this is PAYG billing comes after usage.

Reply:
Okeydokey.

Reply:
Added a dry run metering. So the API is checked but nothing would actually be committed.